GolfExample_TCAPI/assessmenttemplate.html

```diff
@@ -128,18 +141,23 @@
                     },
                     interactionType: questionType,
                     correctResponsesPattern: [
-                        String(correctAnswer)
+                        strCorrectAnswer
                     ]
                 }
             };
 
+            //format numeric response pattern. 
+            if (qObj.definition.interactionType == "numeric"){
+                qObj.definition.correctResponsesPattern[0] = strCorrectAnswer + ":" + strCorrectAnswer;

```

Comment:
No. Well spotted. It should be either:

* ```strCorrectAnswer + "[:]" + strCorrectAnswer;``` or
* ```strCorrectAnswer;```

I prefer the former as it's technically correct with SCORM and the later is more of an allowance in the spec rather than the recommended way. Do you want me to do a commit to fix, or will you fix it as you're in there?